libs/gnutls/Makefile

```diff
@@ -109,19 +109,29 @@ endef
 CONFIGURE_ARGS+= \
 	--enable-shared \
 	--enable-static \
+	--disable-bash-tests \
+	--disable-cxx \
+	--disable-doc \
+	--disable-full-test-suite \
+	--disable-gcc-warnings \
+	--disable-gtk-doc-html \
+	--disable-guile \
+	--disable-ld-version-script \
+	--disable-optional-args \
 	--disable-rpath \
+	--disable-seccomp-tests \
+	--disable-tests \
+	--disable-valgrind-tests \
+	\
 	--disable-libdane \
-	--with-included-unistring \
-	--disable-guile \
-	--disable-nls \
-	--without-idn \
-	--without-zlib \
+	--disable-ssl2-support \
+	--disable-ssl3-support \
 	--enable-local-libopts \
-	--disable-doc \
-	--disable-tests \
+	--without-idn \
 	--with-default-trust-store-dir=/etc/ssl/certs/ \
-	--disable-crywrap \
-	--with-librt-prefix="$(LIBRT_ROOT_DIR)/"
+	--with-included-unistring \
+	--with-librt-prefix="$(LIBRT_ROOT_DIR)/" \
+	--with-pic
```

Comment:
Why is ``` --with-pic``` added?
In general I think it's better to have ARGS listed as upstream

Reply:
Currently both -fPIC and -fpic are passed. --with-pic is default. This just makes it explicit.